src/stories/CardNumberInput.stories.tsx

```
@@ -17,30 +17,50 @@ export default meta;
+
+    return (
+      <CardNumberInput
+        handleCardNumbersChange={handleCardNumbersChange}
+        onSuccessValidate={onSuccessValidate}
+        onSuccessNextInput={onSuccessNextInput}
+      />
+    );
   },
 };
 
 export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)